ReactApp/src/main.jsx

```diff
@@ -2,20 +2,51 @@
  * WordPress dependencies
  */
 import { createRoot, StrictMode } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+import { dispatch } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { store as preferencesStore } from '@wordpress/preferences';
 
 /**
  * Internal dependencies
  */
 import { initializeApiFetch } from './misc/api-fetch-setup';
 import { getGBKit } from './misc/store.js';
+import { getPost } from './misc/Helpers';
 import App from './App.jsx';
 import './index.css';
 
 window.GBKit = getGBKit();
 initializeApiFetch();
 
-createRoot(document.getElementById('root')).render(
-	<StrictMode>
-		<App />
-	</StrictMode>
-);
+function initializeEditor() {
+	const { themeStyles, siteApiRoot } = window.GBKit;
```

Comment:
Given the close proximity to the `window.GBKit` assignment, it likely has little impact, but I suggest we always rely upon the `getGBKit` utility. The consistency in approach might mitigate unexpected, cryptic errors elsewhere if `window.GBKit` is referenced directly. WDYT? 

```suggestion
	const { themeStyles, siteApiRoot } = getGBKit();
```

Reply:
Updated in https://github.com/wordpress-mobile/GutenbergKit/pull/21/commits/c4426ba888c964e4122551a27638ee36a3adbf8f